components/bldc_haptics/include/bldc_haptics.hpp

```diff
@@ -186,25 +186,34 @@ template <MotorConcept M> class BldcHaptics : public BaseComponent {
   /// @return Current position of the haptic motor
   float get_position() const { return current_position_; }
 
-  /// @brief Configure the detents for the haptic motor
-  void update_detent_config(const detail::DetentConfig &config) {
-    std::unique_lock<std::mutex> lk(detent_mutex_);
-    // update the detent center
-    current_detent_center_ = motor_.get().get_shaft_angle();
-
-    if (current_position_ < config.min_position) {
+  /// @brief Sets the current position of the haptic motor to
+  ///        within the bounds of the current detent config
+  void set_position(const int position) {
```

Comment:
Also, might be cleaner code to use std::clamp for this instead of conditional code

Reply:
For sure